runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -138,6 +138,14 @@ initImpl(J9VMThread *currentThread, j9object_t membernameObject, j9object_t refO
 		if (VM_VMHelpers::isTrustedFinalField(fieldID->field, fieldID->declaringClass->romClass)) {
 			flags |= MN_TRUSTED_FINAL;
 		}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		if (
+			vmFuncs->isNameOrSignatureQtype(J9ROMFIELDSHAPE_SIGNATURE(romField)) &&
```

Comment:
formatting, the second line should start with &&


something like
```
if (vmFuncs->isNameOrSignatureQtype(J9ROMFIELDSHAPE_SIGNATURE(romField))
&& vmFuncs->isFlattenableFieldFlattened(fieldID->declaringClass, fieldID->field)
) {
```

Reply:
fixed in 872adf7